src/Forms/TimeField.php

```diff
@@ -337,9 +360,19 @@ public function localisedToISO8601($time)
         $fromFormatter = $this->getFormatter();
         $toFormatter = $this->getISO8601Formatter();
         $timestamp = $fromFormatter->parse($time);
+
+        // Try to parse time without seconds, since that's a valid HTML5 submission format
+        // See https://html.spec.whatwg.org/multipage/infrastructure.html#times
+        if ($timestamp === false && $this->setHTML5(true)) {
```

Comment:
Did you mean to use `$this->getHTML5()` as a condition? This method shouldn't mutate the object.